packages/contract/src/__test__/call-test-contract/src/main.sw

```diff
@@ -40,6 +42,11 @@ impl TestContract for Contract {
     log(value);
     value + 1
   }
+  fn call_external_foo(param: u64, contract_id: b256) -> u64 {
```

Comment:
I was getting errors when I set `contract_id: ContractId` - I'll open an issue to investigate